release-notes-2.0.0-SNAPSHOT.md

```diff
@@ -0,0 +1,13 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
+US397426: Adding date to logs and migrating to SYSTEM_ERR
+
+#### Known Issues
+
+#### Breaking Changes
```

Comment:
When you're modifying Markdown files you need to view them to make sure the formatting is correct.
Look at how this is displaying:
https://github.com/CAFapi/caf-logging/blob/b697d2f23dac2b8d339362a8004d42905a02c654/release-notes-2.0.0-SNAPSHOT.md

The lines are running into each other.

Reply:
Have now made these changes and will submit for re review